test/unit/utils/StreamMessageValidator.test.ts

```diff
@@ -137,12 +145,35 @@ describe('StreamMessageValidator', () => {
             await getValidator().validate(msgWithNewGroupKey)
         })
 
+        it ('rejects unsigned messages when Brubeck validation is required', async () => {
+            getStream = sinon.stub().resolves({
```

Comment:
To make the test case more appropriate, have the `getStream` return a `Stream` where `requireSignedData: false`. (Right now it returns one that has the field set to true, if I understand correctly.)